### PR TITLE
Rszeto toronto

### PR DESCRIPTION
**Pull request by tukaidea**
Please take a look at my top 3

### CODE REVIEW

File: rszeto-toronto.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Places to Eat
+- Canoe
```
Comment (lmyzhao) [inline rszeto-toronto.md]: Canoe is not good!

Comment (lmyzhao) [inline rszeto-toronto.md]: Canoe is not good!

**Comment by lmyzhao** [comment]
@rszeto28 don't waste your money at Canoe!

File: rszeto-toronto.md (hunk 1 above)
Comment (tukaidea) [reply rszeto-toronto.md]: really ? what would you recommend?

**Pull request merged**